src/__tests__/lib/exampleData.js

```diff
@@ -291,6 +292,10 @@ export const streamMessage = (extra?: $Rest<Message, {}>): Message => {
   return deepFreeze({ ...baseMessage, ...extra });
 };
 
+/** Construct a MessagesState from a list of messages. */
+export const makeMessagesState = (messages: Message[]): MessagesState =>
+  objectFromEntries(messages.map(m => [m.id, m]));
```

Comment:
Feedback at https://github.com/zulip/zulip-mobile/pull/4104#discussion_r424735090 resolved by _a)_ stripping the exactness from both `MessagesState` itself (until Flow v0.111.0) and its indexer-property's value type, and _b)_ eliminating the `$FlowFixMe` entirely.